src/expressServer.js

```diff
@@ -168,6 +172,11 @@ io.on('connection', (socket) => {
   console.log('Connected');
   setupSocket(io, socket); // Call your setupSocket function
 
+  socket.on('chat:album', function (data) {
+    const { handleAlbumChat } = require('./live/handleAlbumChat');
+    handleAlbumChat(io, socket, data);
+  });
```

Comment:
We already have this a listener for this event name/channel defined in `live/index.js` we shouldn't need to have it duplicated here. Generally either place is fine to define our listeners, but to try and keep things organized, similar to how we have our HTTP routes defined in the `routes` and `controllers` folders; we'd prefer to have our socket listeners and handlers defined in the `live` folder.

Reply:
Removed